src/MessagesCard/Completed.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import '../MessagesCard/styles/Style-MessagesCard.css'
+
+const Completed = () => (
+    <div id="completed" className="completed">
+            <p>Successful payment</p>
+            <a href="/">Your shopping</a>
```

Comment:
check this indentation

Reply:
I think a more "natural" way to do this is using the Link component

https://reactrouter.com/en/main/components/link

```js
<Link to="/">To Shopping</Link>
```